admin_guide/manage_scc.adoc

```diff
@@ -174,53 +200,81 @@ $ oc edit scc privileged
 ----
 +
 Note that a fully-qualified service account user name is in the form of
-*system:serviceaccount:<namesapce>:<name>*.
+*system:serviceaccount:<namespace>:<name>*.
 
-. Create the registry using `oadm registry --service-account=<name> --mount-host=<dir> ...`
+. Create the registry using:
++
+----
+$ oadm registry --service-account=<name> --mount-host=<dir> ...
+----
 
 === Provide Additional Capabilities
 
-In some cases an image may require capabilities that Docker does not provide out of the box.  In
-this case you may provide the ability to request additional capabilities in the pod spec which
-will be validated against an SCC.
+In some cases, an image may require capabilities that Docker does not provide
+out of the box. You can provide the ability to request additional capabilities
+in the pod specification which will be validated against an SCC.
 
 [IMPORTANT]
 ====
-This allows images to run with elevated capabilities and should be used only if necessary.  You
-should not edit the default restricted SCC to enable additional capabilities.
+This allows images to run with elevated capabilities and should be used only if
+necessary. You should not edit the default *restricted* SCC to enable additional
+capabilities.
 ====
 
-NOTE: When used in conjunction with a non-root user you must also ensure that the file
-that requires the additional capability is granted the capabilities via the `*setcap*` command.  For
-example `*setcap cap_net_raw,cap_net_admin+p /usr/bin/ping*`.
+When used in conjunction with a non-root user, you must also ensure that the
+file that requires the additional capability is granted the capabilities using
+the `setcap` command. For example:
 
-NOTE: If a capability is provided by default in Docker you do not need to modify the pod spec
-to request it.  For example `*NET_RAW*` is provided by default and capabilities should already be
-set on `*ping*`, no special steps should be required to run `*ping*`.
+----
+setcap cap_net_raw,cap_net_admin+p /usr/bin/ping
+----
```

Comment:
Yes, this is fine with me.  After reading through again we might want to call out that this is done in the Dockerfile of the image itself.  I wasn't clear about that originally.